src/index.ts

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+
+const app = express();
+const target: string = process.env.TARGET || 'World, from TypeScript!';
+const responseMessage: string = `Hello ${target}!`;
+
+app.get('/', (_req, res) => {
+  // eslint-disable-next-line no-console
```

Comment:
Would it be better to make our own changes to the linting config or simply 'comment' these out every time?

Reply:
specifically, this is for line 8 up there.

Reply:
I do want us to keep this rule project-wide. When we see `// eslint-disable ... ` messages, it's a signal that the author had overrode the rule for a reason, and then alternatives can be discussed in code review. Or, the author and the reviewer can agree it's a good exception and keep it. 

In this case, the reason is because this is throwaway code we're using to validate the linter and CI, so it's all good. It should stay, and we should continue disallowing console statements across the rest of the project. They're for debugging purposes and should not reach production.